pmm-app/tests/pages/homePage.js

```diff
@@ -1,5 +1,6 @@
-const {I, settingsAPI} = inject();
+const {I} = inject();
 const assert = require('assert');
+const lastCheckRegex = /^(?:(((Jan?|Ma(r?|y)|Jul(y)?|Aug?|Oct?|Dec?)\ 31)|((Jan?|Ma(r?|y)|Apr?|Ju((ly?)|(ne?))|Aug?|Oct?|(Sep(t)?|Nov|Dec)?)\ (0?[1-9]|([12]\d)|30))|(Feb?\ (0?[1-9]|1\d|2[0-8]|(29(?=,\ ((1[6-9]|[2-9]\d)(0[48]|[2468][048]|[13579][26])|((16|[2468][048]|[3579][26])00)))))))\,) (?:[0-1]?[0-9]|[2][1-4]):[0-5]?[0-9]?\s??$/gim;
```

Comment:
What are all those magic numbers, like `[2468][048]|[13579][26]` ? Can we link this regexp to an eventual source (in a comment maybe)?

Reply:
Great point. Sorry, looks like i did not cleanup that regex completely. Added the source link to the comment